src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -287,10 +306,41 @@ export class AddScenarioModal implements OnInit, OnDestroy {
             const controlArray = <FormArray> this.newScenForm.get('regressionRegions.regressions.errors');       
             controlArray.controls[index].get('id').setValue(element.id);
             controlArray.controls[index].get('value').setValue(element.value.toString());
-        });  
+        }); 
         this.cloneParameters = " "; 
     }
 
+    public cloneScenario(){  
+       this.newScenForm.addControl('region', this._fb.control('', Validators.required));
+        this.regions.forEach( (element,index) => {  
+            if (element.id.toString() == this.selectedRegion.id.toString()){
+                this.newScenForm.patchValue({ region: this.regions[index]});
+            }
+        });
+        this.fillModal();
+    }
+
+    public cloneOrEdit(parameter){
+        //make sure in edit mode
+        if (this.editMode == true){
+            this.scen = JSON.parse(JSON.stringify(this.newScenForm.value));
+            //change back to edit if user reselects original core dropdowns
+            if ((parameter == "statGroup" && this.orgStatGrp == this.scen.statisticGroupID) ||
+            (parameter == "regRegion" && this.orgRegRegion == this.scen.regressionRegions.ID) || 
+            (parameter == "regVariable" && this.orgRegVariable == this.scen.regressionRegions.regressions.ID)){
+                this.clone = false;
+                this.edit = true;
+                this._toasterService.clear();
+                this._toasterService.pop('info', 'Info', 'Scenario will be Edited Instead of Cloned');
```

Comment:
Inconsistent capitalization here

Reply:
Do you mean the info's or the 'Scenario will be Edited Instead of Cloned'?

Reply:
I was referring to "Scenario will be Edited Instead of Cloned", as well as the "...Cloned Instead of Edited"